pkg/service/catalog_test.go

```diff
@@ -69,53 +69,56 @@ func init() {
-                    "description":"%s",
-                    "tags":["%s"],
-                    "bindable":%t,
-                    "plan_updateable":%t,
-                    "plans":[
-                        {
-                            "id":"%s",
-                            "name":"%s",
-                            "description":"%s",
-                            "free":%t,
-                            "schemas": {
-                              	"service_instance": {
-                                	"create": {
-                                		"parameters": {
-                                			"$schema": "http://json-schema.org/draft-04/schema#",
-                                			"type": "object",
-                                			"properties": {
-                                				"location": {
-                                					"type": "string",
-                                					"description": "%s"
-                                				},
-                                				"resourceGroup": {
-                                					"type": "string",
-                                					"description": "%s"
-                                                },
-                                                "tags": {
-                                                    "type": "object",
-                                                    "description": "%s",
-                                                    "additionalProperties" : {
-                                                        "type" : "string"
-                                                    }
-                                                }
-                                			}
-                                		}
-                                	}
-                                }
-                            }
-                        }
-                    ]
-                }
-            ]
-        }`
+	testCatalogTemplate := `
+	{
+		"services":[
+			{
+				"name":"%s",
+				"id":"%s",
+				"description":"%s",
+				"tags":["%s"],
+				"bindable":%t,
+				"plan_updateable":%t,
+				"plans":[
+					{
+						"id":"%s",
+						"name":"%s",
+						"description":"%s",
+						"free":%t,
+						"schemas": {
+							"service_instance": {
+								"create": {
+									"parameters": {
+										"$schema": "http://json-schema.org/draft-04/schema#",
+										"type": "object",
+										"properties": {
+											"location": {
+												"type": "string",
+												"description": "%s"
+											},
+											"resourceGroup": {
+												"type": "string",
+												"description": "%s"
+											},
+											"tags": {
+												"type": "object",
+												"description": "%s",
+												"additionalProperties" : {
+													"type" : "string"
+												}
+											}
+										},
+										"additionalProperties": false
+									}
+								}
+							}
+						}
+					}
+        ]
+			}
+		]
+	}
+	`
```

Comment:
All of the above is just white-space fixes, _except_ the addition of `"additionalProperties": false` on line 110.